src/main/resources/version.properties

```diff
@@ -0,0 +1 @@
+projectVersion: APP_VERSION_TOKEN
```

Comment:
file is missing a new line at the end

Reply:
Added new line at the end of version.properties,anet.yml and anet-dictionary.yml files